### PR TITLE
Feature/dx 1047 document transaction problems

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/transaction.md [hunk 1]
```diff
@@ -55,6 +55,49 @@ Content-Type: application/json
...
+        "state": "Failed",
+        "number": 1234567890,
+        "amount": 1000,
+        "vatAmount": 250,
+        "description": "Test transaction",
+        "payeeReference": "AH123456",
+        “failedReason”: “ExternalResponseError”,
+        “failedActivityName”: “Authorize”,
+        “failedErrorCode”: “REJECTED_BY_ACQUIRER”,
+        “failedErrorDescription”: “General decline, response-code: 05",
```
Comment (asbjornu) [inline _includes/transaction.md:84]:
```suggestion
```

File: _includes/transaction.md [hunk 2]
```diff
@@ -55,6 +55,49 @@ Content-Type: application/json
...
+        “activities”: { “id”: “/psp/{{ api_resource }}/payments/{{ page.payment_id }}/transactions/{{ page.transaction_id }}/activities” },
+        “problem”: {
+          “type”: “https://api.payex.com/psp/errordetail/{{ api_resource }}/3DSECUREERROR”,
+          “title”: “Error when complete authorization”,
+          “status”: 400,
+          “detail”: “Unable to complete 3DSecure verification!“,
+          “problems”: [
+          ]
+    }
+    “operations”: [
```
Comment (asbjornu) [inline _includes/transaction.md:95]:
```suggestion
        "isOperational": true,
        "activities": { "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/transactions/{{ page.transaction_id }}/activities" },
        "problem": {
          "type": "https://api.payex.com/psp/errordetail/{{ api_resource }}/3DSECUREERROR",
          "title": "Error when complete authorization",
          "status": 400,
          "detail": "Unable to complete 3DSecure verification!",
          "problems": [
          ]
    }
    "operations": [
```

File: _includes/transaction.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 {% assign api_resource = include.api_resource | default: 'creditcard' %}
 {% assign documentation_section = include.documentation_section %}
 
+{% if documentation_section == "checkout" or documentation_section == "payment-menu" %}
+{% assign documentation_url = documentation_section %}
```
Comment (NullableInt) [inline _includes/transaction.md:5]: Why this?

File: _includes/payment-order-other-features.md [hunk 4]
```diff
@@ -1366,46 +1361,25 @@ Triggered on terminal errors, and when the configuration fails validation.
...
+This is the reason we are explicitly passing `documentation_section=nil` to the
+problems include.
+
+Variable scoping is fixed in the new `{% render %}` tag implemented in Liquid,
+but since no release has been made in Liquid since March 2019, this newly added
+tag can't be used in Jekyll just yet. Once a new release of Liquid has been made
+and Jekyll upgrades to it, we should also upgrade and replace all `include` with
+`render` tags.
+
+https://github.com/Shopify/liquid/pull/1122
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:1380]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5855354&bid=19169599)

**Pull request merged**